src/core/bluetoothdevicemodel.h

```diff
@@ -0,0 +1,91 @@
+/***************************************************************************
+  bluetoothdevicemodel.h - BluetoothDeviceModel
+
+ ---------------------
+ begin                : 20.11.2020
+ copyright            : (C) 2020 by David Signer
+ email                : david (at) opengis.ch
+ ***************************************************************************
+ *                                                                         *
+ *   This program is free software; you can redistribute it and/or modify  *
+ *   it under the terms of the GNU General Public License as published by  *
+ *   the Free Software Foundation; either version 2 of the License, or     *
+ *   (at your option) any later version.                                   *
+ *                                                                         *
+ ***************************************************************************/
+
+#ifndef BLUETOOTHDEVICEMODEL_H
+#define BLUETOOTHDEVICEMODEL_H
+
+#include <QAbstractListModel>
+#include <QtBluetooth/QBluetoothServiceDiscoveryAgent>
+#include <QtBluetooth/QBluetoothServiceInfo>
+#include <QtBluetooth/QBluetoothLocalDevice>
+
+/**
+ * A model that manages the key/value pairs for a ValueMap widget.
```

Comment:
Is this for a valuemap widget? Or a combobox with bluetooth devices?

Reply:
Copied the comment and forgot to edit it :-)